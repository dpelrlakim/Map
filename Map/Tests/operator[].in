newDefault a
[] a 10
[]= a 20 1
[]= a 0 -1
[]= a 11 2
[]= a 30 3
[]= a -10 -2
[]= a 9 8
erase a 0
erase a 9
erase a 20
[] a -10
[] a 0
[] a 9
[] a 10
at a 11
at a 20
at a 30
at a 40
= b a
clear b
erase a 0
erase a 11
erase a 9
erase a 10
