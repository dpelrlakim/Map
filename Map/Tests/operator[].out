a:
0
a: (10,0)
a: (10,0) (20,1)
a: (0,-1) (10,0) (20,1)
a: (0,-1) (10,0) (11,2) (20,1)
a: (0,-1) (10,0) (11,2) (20,1) (30,3)
a: (-10,-2) (0,-1) (10,0) (11,2) (20,1) (30,3)
a: (-10,-2) (0,-1) (9,8) (10,0) (11,2) (20,1) (30,3)
a: (-10,-2) (9,8) (10,0) (11,2) (20,1) (30,3)
a: (-10,-2) (10,0) (11,2) (20,1) (30,3)
a: (-10,-2) (10,0) (11,2) (30,3)
-2
a: (-10,-2) (10,0) (11,2) (30,3)
0
a: (-10,-2) (0,0) (10,0) (11,2) (30,3)
0
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
0
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
2
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
Exception
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
3
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
Exception
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
b: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
a: (-10,-2) (0,0) (9,0) (10,0) (11,2) (30,3)
b:
a: (-10,-2) (9,0) (10,0) (11,2) (30,3)
b:
a: (-10,-2) (9,0) (10,0) (30,3)
b:
a: (-10,-2) (10,0) (30,3)
b:
a: (-10,-2) (30,3)
b:
